### PR TITLE
Fix/remove bed from sample

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
This PR fixes the balsamic bed version not the being actual 

**How to prepare for test**:
- [x] ssh to hasta/clinical-db depending on type of change
- [x] install on stage:
`bash servers/resources/hasta.sclifelab.se/update-cg-stage.sh [THIS-BRANCH-NAME]`
- [x] install on stage:
`bash servers/resources/clinical-db.sclifelab.se/update-clinical-api-stage.sh [THIS-BRANCH-NAME]`

**How to test bed version not being set on order submit**:
- [x] login to the orderportal-stage
- [x] submit balsamic order
- [x] open clinical-api-stage samples view and find submitted sample

**Expected test outcome**:
- [x] sample has no bed-version connected to it (bed-version is not part of sample anymore)

**How bed version being fetched from lims**:
- [x] login to hasta
- [x] us
- [x] `cg workflow balsamic config-case legalgull --dry-run`

**Expected test outcome**:
- [x] output contains -p flag with something that looks like a complete bed-path
- [x] Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by @adrosenbaum 
- [x] tests executed by @patrikgrenfeldt 
- [x] "Merge and deploy" approved by @patrikgrenfeldt 
Thanks for filling in who performed the code review and the test!

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [x] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: scripts/alter-bed-version.sql [hunk 1]
```diff
@@ -0,0 +1,22 @@
+-- show data to be removed
```
Comment (adrosenbaum) [inline scripts/alter-bed-version.sql]: remove line 1-6

Comment (patrikgrenfeldt) [reply scripts/alter-bed-version.sql]: removed

**Comment by patrikgrenfeldt** [comment]
Test result OP:
OP has bed:
image
sample has not:
image
👍

**Comment by patrikgrenfeldt** [comment]
@adrosenbaum Sorry for the late code change. Testing indicated that the balsamic case-config command was broken

**Comment by patrikgrenfeldt** [comment]
Test result cli case-config: 
image
👍

File: cg/cli/workflow/balsamic/base.py [hunk 2]
```diff
@@ -45,13 +45,13 @@ def balsamic(context, case_id, priority, email, target_bed):
...
     context.obj["gzipper"] = gzip
+    context.obj["lims_api"] = lims.LimsAPI(context.obj)
     scout_api = scoutapi.ScoutAPI(context.obj)
-    lims_api = lims.LimsAPI(context.obj)
     tb_api = tb.TrailblazerAPI(context.obj)
     deliver = DeliverAPI(
         context.obj,
         hk_api=context.obj["hk_api"],
-        lims_api=lims_api,
+        lims_api=context.obj["lims_api"],
```
Comment (adrosenbaum) [inline cg/cli/workflow/balsamic/base.py]: could the lims-api be created under the `workflow` group command instead if more workflows will/are using it?

Comment (patrikgrenfeldt) [reply cg/cli/workflow/balsamic/base.py]: Let's keep it specific for now to keep it more readable (hopefully)

**Comment by patrikgrenfeldt** [comment]
deployed on hasta and clinical-db

**Pull request merged**